mu-plugins/blocks/language-suggest/src/front.js

```diff
@@ -0,0 +1,23 @@
+const init = () => {
+	const container = document.querySelector( '.wp-block-wporg-language-suggest' );
+
+	if ( ! container ) {
+		return;
+	}
+
+	// Construct the URL without protocol
+	const path = window.location.host + window.location.pathname;
+
+	fetch( `/lang-guess/lang-guess-ajax.php?uri=${ encodeURIComponent( path ) }` )
```

Comment:
This was repurposed from the original code but it would probably be better if it wasn't relative.